client/question/components/LearnerViewDirective.js

```diff
@@ -105,9 +105,12 @@ tie.directive('learnerView', [function() {
                     </ui-codemirror>
                   </div>
                 </div>
-                <select class="tie-select-menu" name="lang-select-menu" ng-class="{'night-mode': isInDarkMode}">
+                <select ng-if="SERVER_URL" class="tie-select-menu" name="lang-select-menu" ng-class="{'night-mode': isInDarkMode}">
                   <option value="Python" selected>Python</option>
                 </select>
+                <button ng-if="!SERVER_URL" class="tie-python-primer tie-button" ng-class="{'night-mode': isInDarkMode}">
+                  <a class="tie-primer-link" ng-class="{'night-mode': isInDarkMode}" target="_blank" ng-href="../docs/py-primer-{{isInDarkMode === true ? 'dark' : 'light'}}.html">New to python?</a>
```

Comment:
Could you extract this into $scope.getPythonPrimerUrl()?

Also, note that you don't need the `=== true` bit. `isInDarkMode ? 'dark' : 'light'` should suffice.